src/Microsoft.Management.Deployment/PackageCatalogReference.cpp

```diff
@@ -64,8 +63,14 @@ namespace winrt::Microsoft::Management::Deployment::implementation
                 {
                     auto catalog = m_compositePackageCatalogOptions.Catalogs().GetAt(i);
                     winrt::Microsoft::Management::Deployment::implementation::PackageCatalogInfo* catalogInfoImpl = get_self<winrt::Microsoft::Management::Deployment::implementation::PackageCatalogInfo>(catalog.Info());
-                    ::AppInstaller::Repository::SourceDetails sourceDetails = catalogInfoImpl->GetSourceDetails();
-                    std::shared_ptr<::AppInstaller::Repository::ISource> remoteSource = ::AppInstaller::Repository::OpenSourceFromDetails(sourceDetails, progress).Source;
+                    
+                    ::AppInstaller::Repository::SourceDetails details = catalogInfoImpl->GetSourceDetails();
+                    // If the AdditionalPackageCatalogArguments value has been explicitly set on the composite, then override the value on each individual catalog.

```

Comment:
I think we should fail to set the value on the Reference rather than support this.  If the caller truly wants to pass it to all of the other sources they can do so programmatically easily enough.